enclave/src/utils.rs

```diff
@@ -101,11 +123,28 @@ pub fn read_file(filepath: &str) -> SgxResult<Vec<u8>> {
 	}
 }
 
-// FIXME: think about how statevec should be handled in case no counter exist such that we
-// only need one read function. Maybe search and init COUNTERSTATE file upon enclave init?
-pub fn read_counterstate(filepath: &str) -> SgxResult<Vec<u8>> {
+pub fn read_counterfile() -> SgxResult<Vec<u8>> {
```

Comment:
I really don't like the naming. While in our demo case it actually is a counter, this doesn't apply in general. I think this should be named `read_state_from_file()`